### PR TITLE
Bumped ts-eslint version to enforce access modifier

### PR DESCRIPTION
**Pull request by rohan-bes**
- Added modifiers to all existing ts classes
- Fixed up a couple linting errors here and there

### CODE REVIEW

File: packages/admin-panel-server/src/routes/FetchReportPreviewDataRoute.ts [hunk 1]
```diff
@@ -95,16 +95,18 @@ export class FetchReportPreviewDataRoute extends Route<FetchReportPreviewDataReq
         draftDashboardItemValidator,
         draftReportValidator,
       );
-    } else if (vizType === VIZ_TYPE_PARAM.MAP_OVERLAY) {
+    }
+
+    if (vizType === VIZ_TYPE_PARAM.MAP_OVERLAY) {
```
Comment (rohan-bes) [inline packages/admin-panel-server/src/routes/FetchReportPreviewDataRoute.ts:100]: Linter complains about `else if` after a `return` as `if` would suffice

File: packages/api-client/src/connections/EntityApi.ts [hunk 2]
```diff
@@ -37,7 +35,7 @@ const recurseFilter = (
     const value = filter.comparisonValue;
     filterArray.push([
       nestedKeys,
-      // @ts-ignore
```
Comment (rohan-bes) [inline packages/api-client/src/connections/EntityApi.ts:40]: `@ts-ignore` gives a linting error, but `ts-expect-error` doesn't as long as we provide and explanation. So we should always use that 👍

File: packages/entity-server/src/routes/hierarchy/relationships/ResponseBuilder.ts [hunk 3]
```diff
@@ -137,23 +146,12 @@ export class ResponseBuilder {
     return formattedEntitiesByCode;
   }
 
-  isDefaultFilter = (filter: EntityFilter) => {
```
Comment (rohan-bes) [inline packages/entity-server/src/routes/hierarchy/relationships/ResponseBuilder.ts:140]: Cosmetic but in this case I felt the function made a bit more sense outside the class (as it didn't rely on class members and is kind of generic)

File: packages/data-lake-api/src/DataLakeDatabase.ts [hunk 4]
```diff
@@ -3,9 +3,9 @@
  * Copyright (c) 2017-2020 Beyond Essential Systems Pty Ltd
  */
 
-// @ts-ignore
+// @ts-expect-error pg has not types
```
Comment (edmofro) [inline packages/data-lake-api/src/DataLakeDatabase.ts:6]:
```suggestion
// @ts-expect-error pg has no types
```

File: packages/psss-server/src/routes/reports/ConfirmWeeklyReportRoute.ts [hunk 5]
```diff
@@ -3,8 +3,10 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (edmofro) [inline packages/psss-server/src/routes/reports/ConfirmWeeklyReportRoute.ts:6]: Why was this needed?

**Review by edmofro — APPROVED** [review]
Thanks for doing this chore! And cleaning up a few bits and pieces along the way, including some of those wasteful extra lines between fields 😀

File: packages/psss-server/src/routes/reports/ConfirmWeeklyReportRoute.ts (hunk 5 above)
Comment (rohan-bes) [reply packages/psss-server/src/routes/reports/ConfirmWeeklyReportRoute.ts]: Good question! I added `lodash.groupby` to the package dependencies instead 👍
